packages/gatsby-source-nacelle/src/source-nodes/index.js

```diff
@@ -6,6 +6,7 @@ const {
   replaceKey,
   findNestedImages
 } = require('../utils');
+const findNestedReferences = require('../utils/linkNestedContentReferences');
```

Comment:
Could we add this to `packages/gatsby-source-nacelle/src/utils/index.js` and import it alongside the other utils?

Reply:
Yeah. Done!